recipes/libglvnd/meta.yaml

```
@@ -0,0 +1,152 @@
+{% set version = "1.7.0" %}
+
+package:
+  name: libglvnd-split
+  version: {{ version }}
+
+source:
+  url: https://gitlab.freedesktop.org/glvnd/libglvnd/-/archive/v{{ version }}/libglvnd-v{{ version }}.tar.bz2
+  sha256: d0e1925a3c9aee0143b8c181ac31d5637c8faa081759c277b8e16c7075612c11
+  patches:
+    - libglvnd_look_for_egl_icd_in_system_location.patch
+
+build:
+  skip: true  # [not linux]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - meson
+    - pkg-config
+    - ninja
+  host:
+    - xorg-libx11
+    - xorg-libxext
+    - xorg-xextproto
+    - xorg-glproto
+
+outputs:
+  - name: libglvnd
+    script: build_libglvnd.sh
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ stdlib('c') }}
+        - meson
+        - pkg-config
+        - ninja
+      host:
+        - xorg-libx11
+        - xorg-libxext
+        - xorg-xextproto
+        - xorg-glproto
+    test:
+      commands:
+        - test -f $PREFIX/lib/libEGL.so.1
+        - test -f $PREFIX/lib/libGL.so.1
+        - test -f $PREFIX/lib/libGLESv1_CM.so.1
+        - test -f $PREFIX/lib/libGLESv2.so.2
+        - test -f $PREFIX/lib/libGLX.so.0
+        - test -f $PREFIX/lib/libOpenGL.so.0
+        - test -f $PREFIX/lib/libGLdispatch.so.0
```

... where `libraries` looks like:
```suggestion
        {% set libraries = ["EGL", "GL", "GLESv1_CM", "GLESv2", "GLX", "OpenGL", "GLdispatch"] %}
        {% for each_lib in libraries %}
        - test -f $PREFIX/lib/lib{{ each_lib }}.so
        {% endfor %}
```
I think that testing the exact SOVER of the library is not important, but if this is desired, we could obviously adjust the list to be something like `libraries = ["EGL.so.1", ...` and adapt as necessary.

---

Lets keep the so version check.

I personally dislike the jinja indirection in the recipes. It makes them "not simple".

The so version also ensures that the run export is working as expected.

---

```suggestion
        # shared libraries
        {% set libraries = [
            "EGL.so.1", "GL.so.1", "GLESv1_CM.so.1", "GLESv2.so.2",
            "GLX.so.0", "OpenGL.so.0", "GLdispatch.so.0"
        ] %}
        {% for each_lib in libraries %}
        - test -f $PREFIX/lib/lib{{ each_lib }}.so
        {% endfor %}
```